### ISSUE TITLE
Could you clarify what the computed values are for calc(infinity*1px) and calc(NaN*1px)?

### ISSUE THREAD

**Issue by simevidas** [DevSDK/calc-infinity-and-NaN#2]
In the README, the `<length>` example says that `calc(infinity*1px)` evaluates to the maximum value, but then it says the same thing for `calc(NaN*1px)`, so what’s the difference?

<img width="866" alt="Screen Shot 2020-12-19 at 5 01 35 PM" src="https://user-images.githubusercontent.com/716405/102693586-feb8b480-421b-11eb-89ff-7f77599b0b9b.png">

**Comment by DevSDK** [comment]
Thanks for the question.
According to the [spec ](https://www.w3.org/TR/css-values-4/#top-level-calculation), It says "If a top-level calculation (a math function not nested inside of another math function) would produce a value whose numeric part is NaN, it instead act as though the numeric part is +∞. "
Therefore NaN will be calculated as positive infinity. 
I'll refer there thanks !

**Comment by yisibl** [comment]
![image](https://user-images.githubusercontent.com/2784308/145763478-1597e5e3-d68a-4466-9e3e-7b205f0da998.png)

The current implementation in Chrome calculates the value of `1.67772e+07`, this is not the same as in the documentation?

**Comment by DevSDK** [comment]
@yisibl 

Sorry for the late repose.

Thanks for sharing that :)

Can I ask to get the code and system information you've tested for?

That's strange... 

I've compiled and run on my macbook pro 
![image](https://user-images.githubusercontent.com/18409763/146013629-ef56867b-b7d0-4af2-8430-fc0e3cbf9788.png)

And I get the value close to the document.

![image](https://user-images.githubusercontent.com/18409763/146013542-a95d3416-fb08-436e-8a48-0529c8618241.png)
![image](https://user-images.githubusercontent.com/18409763/146013586-e939b43f-5406-47c1-8d90-ec31284ce4bf.png)

I guess that is a system or architecture difference but I'm investigating what exactly makes the difference. So it may need time.

I'll let you know when I found what is the difference. (With the update the document)

Thank you again!

**Comment by yisibl** [comment]
@DevSDK Thanks!
![image](https://user-images.githubusercontent.com/2784308/146304888-e392cc07-4b53-43d0-a170-9516b7e91308.png)

**Comment by yisibl** [comment]
`z-index: calc(infinity * 1)` What is the maximum value of z-index on your computer?
![image](https://user-images.githubusercontent.com/2784308/146309321-44822a19-67aa-4581-b30e-94bb08d3b8a4.png)

**Comment by DevSDK** [comment]
@yisibl 
It is 2147483647 which is INT_MAX.

**Comment by DevSDK** [comment]
Hi @yisibl.

I've found what makes difference.

In a nutshell, It is a resolution difference (or dpi)  that will make a different zoom factor.

I've confirmed length computed value changed to `1.67772e+07` when I move the window to my built-in display.

I've updated the explainer.md file the formula of the maximum computed value (INT_MAX / kFixedPointDenominator / zoom factor ≈ 3.35544e+07 if zoom factor is 1).

Thanks for your asking!
